src/content/docs/release-notes/new-relic-browser-release-notes/browser-agent-release-notes/browser-agent-v1.239.1.mdx

```diff
@@ -0,0 +1,50 @@
+
+
+
+
+---
+subject: Browser agent
+releaseDate: "2023-09-01”
+version: 1.239.1
+features: ["Add best effort to detect Ajax events before instantiation","Add mode to enable agent to not harvest until user consent", "Remove allow_bfcache flag", "Remove old supportability metrics"]
+bugs: ["Fix issue with leaking event listeners","Issues with types resolution", "Fix internal release process"]
+security: []
+---
+
+## v1.239.1
+
+### Features
+
+#### Add best effort to detect Ajax events before instantiation
+Add a "best effort" at detecting ajax events that may have happened on the page before the agent was instantiated by utilizing the ResourceTiming browser API.
+
+#### Add mode to enable agent to not harvest until user consent
+Add ability for customers to wait to start harvesting. This behavior will utilize a configuration setting in tandem with an API call to trigger features to begin harvesting.
+
+#### Remove allow_bfcache flag
+Removes unused code which still relied on the window unload event. Going forward, the only method the agent will use to monitor potential end of page session will be visibility change in accordance with modern web practice.
+
+#### Remove old supportability metrics
+Removing supportability metrics that are no longer needed including polyfill detection, bfcache usage, body and query bytes sent count, and custom attributes size.
+
+### Bug Fixes
+
+#### Fix issue with leaking event listeners
+Send buffered session trace data in certain cases when it's being discarded at the tail end of a captured session.
```

Comment:
```suggestion
Fixing an issue where session timeout would result in rewrapping global event handlers. For instances where the site would be open for long periods of time, such as kiosks, this would eventually result in a degradation in event handler performance.
```